addons/mail/models/discuss/discuss_channel.py

```diff
@@ -915,13 +915,20 @@ def channel_get(self, partners_to, pin=True):
             channel._broadcast(partners_to)
         return channel
 
-    def channel_fold(self, state=None, state_count=0):
+    def _channel_fold(self, state=None, state_count=0):
```

Comment:
default values should be removed from all methods, they need to be provided to avoid inconsistencies and race conditions